### PR TITLE
RT-009-010 Add layout app(pallete,typo,font,styles)

### PR DESCRIPTION
**Pull request by dominikszczuka**
I added a palette with colors and object typography with different font options. I changed all styles for app. Now I use CSS on components files. I deleted package for bem css modules.

### CODE REVIEW

File: src/Components/ToDoItem/ToDoItem.tsx [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
-import { default as toDoItemStyles } from "./ToDoItem.module.scss";
-import bemCssModule from "bem-css-modules";
 import { TodoTypes } from "constants/types/TodoTypes";
-import { useTranslation } from "react-i18next";
-
-const style = bemCssModule(toDoItemStyles);
+// import { useTranslation } from "react-i18next";
 
 export interface TodoItemProps {
   todo: TodoTypes;
```
Comment (MRadek55) [inline src/Components/ToDoItem/ToDoItem.tsx]: You should keep interfaces names in singular form

File: src/Components/CustomButton/CustomButton.tsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React from "react";
 import { css, StyleDeclaration, StyleSheet } from "aphrodite";
 import Icon, { IconTypes } from "components/Icon/Icon";
-
+import { typography } from "styles/typography";
+import { palette } from "styles/palette";
```
Comment (MRadek55) [inline src/Components/CustomButton/CustomButton.tsx]: Create src/styles/index.ts and export there typography and palette, then you can make one import -> {typography, palette} from 'styles'

File: src/Components/CustomButton/CustomButton.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { css, StyleDeclaration, StyleSheet } from "aphrodite";
 import Icon, { IconTypes } from "components/Icon/Icon";
 import { typography } from "styles/typography";
-import { palette } from "styles/palette";
+//import { palette } from "styles/palette";
```
Comment (MRadek55) [inline src/Components/CustomButton/CustomButton.tsx]: remove unnecessary code + remember about  import from index (typography, palette)

File: src/Components/Form/Form.tsx [hunk 4]
```diff
@@ -1,32 +1,51 @@
 import React from "react";
 import { StyleSheet, css } from "aphrodite";
 import CustomInput from "components/CustomInput/CustomInput";
+import { palette } from "styles/palette";
+import { typography } from "styles/typography";
```
Comment (MRadek55) [inline src/Components/Form/Form.tsx]: the same

File: src/Components/Form/Form.tsx [hunk 5]
```diff
@@ -1,32 +1,51 @@
...
-    <div className={css(styles.wrapper)}>
-      <h2>Add new tasks</h2>
-      <form>
+    <div className={css(styles.formWrapper)}>
+      <h2 className={css(typography.logoFont, styles.formTitle)}>
+        {t("add-new-task")}
+      </h2>
+      <form className={css(styles.form)}>
         <CustomInput
           placeholder="Write title your task..."
```
Comment (MRadek55) [inline src/Components/Form/Form.tsx]: i18n

File: src/Components/Form/Form.tsx [hunk 6]
```diff
@@ -1,32 +1,51 @@
...
           type="text"
-          disabled={true}
-          customStyle={styles.inputText}
+          disabled={false}
+        />
+        <CustomInput
+          placeholder="Write description"
```
Comment (MRadek55) [inline src/Components/Form/Form.tsx]: i18n

File: src/Components/Form/Form.tsx [hunk 7]
```diff
@@ -1,32 +1,51 @@
...
 
 const Form = () => {
+  const { t } = useTranslation();
+
   return (
-    <div className={css(styles.wrapper)}>
-      <h2>Add new tasks</h2>
-      <form>
+    <div className={css(styles.formWrapper)}>
+      <h2 className={css(typography.logoFont, styles.formTitle)}>
```
Comment (MRadek55) [inline src/Components/Form/Form.tsx]: I think that you can spread typography.logoFont in your formTitle. It should be cleaner.

File: src/Components/Header/Header.tsx [hunk 8]
```diff
@@ -1,14 +1,23 @@
...
+import { typography } from "styles/typography";
 const Header = () => {
   const { t } = useTranslation();
 
   return (
-    <header className="header">
-      <p className="header__logo">{t("title-app")} </p>
-    </header>
+    <p className={css(typography.logoFont, styles.logoText)}>
+      {t("title-app")}{" "}
```
Comment (MRadek55) [inline src/Components/Header/Header.tsx]: It is on purpose ? -> {" "}

File: src/Components/Nav/Nav.tsx [hunk 9]
```diff
@@ -4,6 +4,9 @@ import NavLink from "../NavLink/NavLink";
 
 import { useTranslation } from "react-i18next";
 import { LinkElement } from "constants/types/LinkElement";
+import { StyleSheet, css } from "aphrodite";
+import { palette } from "styles/palette";
+import { typography } from "styles/typography";
```
Comment (MRadek55) [inline src/Components/Nav/Nav.tsx]: You know

File: src/Components/Nav/Nav.tsx [hunk 10]
```diff
@@ -24,9 +27,21 @@ const Nav = () => {
 
   return (
     <nav>
-      <ul>{renderLinksItems()}</ul>
+      <ul className={css(typography.bigFont, style.navigation)}>
```
Comment (MRadek55) [inline src/Components/Nav/Nav.tsx:29]: Also you can spread into styleS.navigation

File: src/Components/ToDoItem/ToDoItem.tsx [hunk 11]
```diff
@@ -10,10 +15,57 @@ const TodoItem: React.FC<TodoItemProps> = ({ todo }: TodoItemProps) => {
...
-      <p>{todo.title}</p>
+    <div className={css(styles.taskWrapp)}>
+      <div className={css(styles.taskHeader)}>
+        <p className={css(typography.bigFont, styles.taskTitle)}>
+          {todo.title}
+        </p>
+        <Icon iconName={Icons.faGraduationCap} iconSize="2x" />
+      </div>
+      <p className={css(typography.normalFont, styles.taskDescription)}>
+        Description:
```
Comment (MRadek55) [inline src/Components/ToDoItem/ToDoItem.tsx]: i18n

File: src/Components/ToDoItem/ToDoItem.tsx [hunk 12]
```diff
@@ -10,10 +15,57 @@ const TodoItem: React.FC<TodoItemProps> = ({ todo }: TodoItemProps) => {
...
+    justifyContent: "space-around",
+    width: "370px",
+    height: "220px",
+    margin: "10px 5px",
+    padding: "6px",
+    backgroundColor: `${palette.blue}`,
+    border: `2px solid ${palette.darkBlue}`,
+    borderRadius: "5px 5px 5px 5px",
+    color: `${palette.white}`,
+    boxShadow: `2px 2px 4px ${palette.darkGray}`,
```
Comment (MRadek55) [inline src/Components/ToDoItem/ToDoItem.tsx]: If you use shadow in many places and it is look the same you should move these shadow styles into typography.ts / styles.ts

File: src/Screens/HomeScreen/HomeScreen.tsx [hunk 13]
```diff
@@ -8,6 +8,9 @@ import { useDispatch } from "react-redux";
 import Form from "components/Form/Form";
 import CustomButton from "components/CustomButton/CustomButton";
 import { Icons } from "constants/enums/Icons";
+import { StyleSheet, css } from "aphrodite";
+import { palette } from "styles/palette";
+import { typography } from "styles/typography";
```
Comment (MRadek55) [inline src/Screens/HomeScreen/HomeScreen.tsx]: index import

File: src/Screens/HomeScreen/HomeScreen.tsx [hunk 14]
```diff
@@ -31,11 +35,45 @@ const HomeScreen = () => {
...
     </LayoutWrapper>
   );
 };
 
+const styles = StyleSheet.create({
+  customBtnStyles: {
+    color: `${palette.white}`,
+    background: `${palette.lightRed}`,
+    border: `2px solid ${palette.darkRed}`,
+    boxShadow: `2px 2px 3px ${palette.black}`,
```
Comment (MRadek55) [inline src/Screens/HomeScreen/HomeScreen.tsx]: shadow

File: src/index.css [hunk 15]
```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f0f9f9;
```
Comment (MRadek55) [inline src/index.css]: palette

**Review by WiktorMonka — APPROVED** [review]

File: src/styles/index.ts [hunk 16]
```diff
@@ -0,0 +1,4 @@
+import { palette, shadow } from "./palette";
+import { typography } from "./typography";
+
+export { palette, shadow, typography };
```
Comment (MRadek55) [inline src/styles/index.ts]: try only this:

export * from './palette';
export * from './typography';

and move shadow into something like commonStyles.tsx

**Review by MRadek55 — APPROVED** [review]

**Pull request merged**